python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -367,16 +368,18 @@ def from_package_module(
         Returns:
             AssetGroup: An asset group with all the assets in the package.
         """
-        assets: Set[AssetsDefinition] = set()
-        source_assets: Set[SourceAsset] = set()
+        assets: Dict[AssetsDefinition, None] = OrderedDict()
```

Comment:
I think it's worth including a short comment on why we use an `OrderedDict` here.